plugins/msteams/README.md

```diff
@@ -0,0 +1,60 @@
+Microsoft Teams Plugin
+==============
+
+Send Microsoft Teams messages for new alerts.
+
+If you have the Python Jinja2 package installed you can customize the
+Microsoft Teams message format.
+
+For help, join [![Gitter chat](https://badges.gitter.im/alerta/chat.png)](https://gitter.im/alerta/chat)
+
+Installation
+------------
+
+Clone the GitHub repo and run:
+
+    $ python setup.py install
+
+Or, to install remotely from GitHub run:
+
+    $ pip install git+https://github.com/alerta/alerta-contrib.git#subdirectory=plugins/msteams
+
+Note: If Alerta is installed in a python virtual environment then plugins
+need to be installed into the same environment for Alerta to dynamically
+discover them.
+
+Configuration
+-------------
+
+Add `msteams` to the list of enabled `PLUGINS` in `alertad.conf` server
+configuration file and set plugin-specific variables either in the
+server configuration file or as environment variables.
+
+```python
+PLUGINS = ['msteams']
+MS_TEAMS_WEBHOOK =  'https://outlook.office.com/webhook/.../IncomingWebhook/.../...'
```

Comment:
Update docs to use`MS_TEAMS_URL` or `MS_TEAMS_WEBHOOK_URL` depending on what you chose in the plugin.

Reply:
Change the README to reflect use of `MS_TEAMS_WEBHOOK_URL` now.